docs/series/README.md

```diff
@@ -1,6 +1,52 @@
----
-title: "Howto Series"
----
+# Howto Series: AlmaLinux Tutorials 🚀
 
-# Series Page
+Welcome to our Howto series on AlmaLinux! In this series, we provide comprehensive guides and tutorials to help you learn and master various aspects of AlmaLinux. Whether you are interested in Nginx or system-related topics, we've got you covered.
 
+## Nginx Series
+
+The Nginx series focuses on all things related to Nginx web server. From installation to configuration, security, and optimization, you will find step-by-step guides to help you deploy and manage a powerful web server on AlmaLinux.
+
+Browse the articles in the Nginx series:
+
+- [A01](nginx/NginxSeriesA01.md) ❯ Beginner's Guide
+- [A02 R8](nginx/NginxSeriesA02R8.md) ❯ AlmaLinux 8.x Installation Examples
+- [A02 R9.1](nginx/NginxSeriesA02R91.md) ❯ AlmaLinux 9.1 Installation Examples
+- [A02 R9.2](nginx/NginxSeriesA02R92.md) ❯ AlmaLinux 9.2 Installation Examples
+- [A03](nginx/NginxSeriesA03.md) ❯ Default Configuration Guide
+- [A04](nginx/NginxSeriesA04P1.md) ❯ Secure Nginx Deployment
+
+## System Series
+
+**The vision:** (In near future) The System series covers a wide range of system-related topics on AlmaLinux. This guides will help you enhance your system administration skills and ensure the stability and security of your AlmaLinux environment.
```

Comment:
**The vision:** (In the near future) The System series covers a wide range of system-related topics on AlmaLinux. These guides will help you enhance your system administration skills and ensure the stability and security of your AlmaLinux environment.